.circleci/config.yml

```diff
@@ -63,6 +44,18 @@ jobs:
           name: 'Code coverage'
```

Comment:
Should code coverage be run here? Or broken out into its own job? Otherwise, we'll run this across multiple versions, which seems unnecessary?

Reply:
Updated code coverage to be its own job.